packages/vsx-registry/src/browser/vsx-extensions-widget.ts

```diff
@@ -52,32 +54,65 @@ export class VSXExtensionsWidget extends SourceTreeWidget {
     protected readonly extensionsSource: VSXExtensionsSource;
 
     @postConstruct()
-    protected override init(): void {
+    protected override async init(): Promise<void> {
```

Comment:
Don't make the post-construct method async: Inversify 5 doesn't wait for it.

Please use the `Promise.then` API instead.